contracted-contributor.md

```diff
@@ -0,0 +1,139 @@
+## Title
+
+_Contracted Contributor_ (aka _Free Electron_)
+
+## Context
+
+Company A has started an InnerSource initiative. One of the goals for this
+initiative is to foster innovation by allowing every associate to voluntarily
+contribute to InnerSource projects, regardless of topic and business unit.
+
+Top level management is on board with InnerSource and is supporting the
+initiative to introduce it. For them, the InnerSource initiative is just one of
+many initiatives to foster innovation and efficiency, though. They are funding
+InnerSource with money and capacity for community leaders and are largely
+giving autonomy as for how the budget is spent. They are also limiting the
+breadth and duration of the initiative and partake in periodic reviews until
+there is proof that it yields the expected results (see [Review
+Committee](https://github.com/paypal/InnerSourcePatterns/blob/master/review-committee.md)).
+Top level management has announced their support for InnerSource on various
+company internal meetings.
+
+However, top level management has not yet empowered or incentivised mid-level
+managers to allow or even motivate their employees to participate in
+cross-divisional InnerSource activities. In addition to that, the capacity of
+every associate is usually allocated to non InnerSource projects for 100 % of
+their working time. Cross organizational collaboration is not yet the norm and
+line managers usually do not have targets outside of their own organisation.
+Contributions to InnerSource projects are expected to be made during working
+hours, not during free time.  
+
```

Comment:
Mention that the solution mentioned below might be more applicable to larger companies with the ability (and a history) of setting up central initiatives supporting their many BUs. Company culture probably also plays a role.